src/pages/workspace/WorkspaceInvitePage.tsx

```diff
@@ -76,11 +68,12 @@ function WorkspaceInvitePage({route, betas, invitedEmailsToAccountIDsDraft, poli
     });
 
     useEffect(() => {
-        return () => {
+        const unsubscribe = navigation.addListener('beforeRemove', () => {
             Member.setWorkspaceInviteMembersDraft(route.params.policyID, {});
-        };
```

Comment:
From the Bug Zero Checklist of #52479: this change raises an issue where the draft value is being cleared. The updated solution for this is to clear the draft value only when starting a new flow.